### ISSUE TITLE
Add css prefixer

### ISSUE THREAD

**Issue by gcmznt** [Objectway/ow-starterkit#6]
https://github.com/pixnbits/gulp-css-prefix

`npm install --save-dev gulp-css-prefix`
## gulp

`var cssPrefix = require('gulp-css-prefix');`

```
// after sass pipe
.pipe(cssPrefix({
    'parentClass': env.namespaceCSS
}))
```

**Comment by genoma** [comment]
:+1:

**Comment by genoma** [comment]
It adds an average of 3.5 seconds to the compilation of a pretty basic Sass :exclamation:

**Comment by gcmznt** [comment]
we can execute this task only if there is a configuration that enable it with gulpif

**Comment by genoma** [comment]
We should yes :+1:

**Comment by genoma** [comment]
Fixed in fce62d0c499ec337b0761459c5e00ff5d7482461
